core/src/main/resources/hudson/model/User/builds.properties

```diff
@@ -20,3 +20,4 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 title=Builds for {0}
+disclaimer=This does not include stages from pipeline jobs.
```

Comment:
Why the different wording than the other ones?

Would be nice to define this message in only one place.